
export const Projects = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const ProjectCreate = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const ProjectDetails = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const ProjectEdit = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const ProjectDelete = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}

export const ProjectLatest = async (req, res, next) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            success: false,
            message: ["Internal Server Error". error.message]
        })
    }
}